### PR TITLE
feature/APPEALS-38446-26725-36789 - Update to Rails 6.0 (release)

### PR DESCRIPTION
**Pull request by jcroteau**
Combines change sets from
- https://github.com/department-of-veterans-affairs/caseflow/pull/20539
- https://github.com/department-of-veterans-affairs/caseflow/pull/20993
- https://github.com/department-of-veterans-affairs/caseflow/pull/20298

### CODE REVIEW

**Review by sbashamoni — APPROVED** [review]

**Review by craigrva — APPROVED** [review]

File: config/environments/production.rb [hunk 1]
```diff
@@ -58,9 +56,9 @@
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
 
-  # Use a real queuing backend for Active Job (and separate queues per environment)
+  # Use a real queuing backend for Active Job (and separate queues per environment).
   # config.active_job.queue_adapter     = :resque
-  # config.active_job.queue_name_prefix = "caseflow_certification_#{Rails.env}"
+  # config.active_job.queue_name_prefix = "caseflow_certification_production"
```
Comment (craigrva) [inline config/environments/production.rb:61]: I'm not sure why this change is being made. Prodtest uses the same config file as Production, which will cause issues if this line ever becomes un-commented

Comment (jcroteau) [reply config/environments/production.rb:61]: There are many such changes resulting from `bin/rails app:update`.

If you like, you can review the commits individually at https://github.com/department-of-veterans-affairs/caseflow/pull/20993/commits, namely those commits with message like `🚧 'bin/rails app:update' - Apply changes to '<filename here>'`

Comment (jcroteau) [reply config/environments/production.rb:61]: >Prodtest uses the same config file as Production, which will cause issues if this line ever becomes un-commented

Also, `Rails.env` (not to be confused with `Rails.deploy_env`) is `production` in the prodtest environment, so this does not really change anything anyways.
